audb/core/load.py

```diff
@@ -403,6 +404,43 @@ def job(cache_root: str, file: str):
     return missing_tables
 
 
+def _load_header(
```

Comment:
Moved the core functionality of `load_header()` to this helper function that we can access it by other functions without accessing the lock again.